src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -496,19 +496,21 @@ public void testPanelsCollapseExpand() {
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToCollapse();
         assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(false));
+        assertTrue(resultsPage.areResultsHidden());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToExpand();
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
```

Comment:
I have made this change as it was suggested in PR #4664 and the thing I am fixing here is to solve a similar problem face in PR #4664 @wkurniawan07 

Should I change it back to start with verify or keep this as it is?